### PR TITLE
added turtl cattle template

### PR DESCRIPTION
**Pull request by fredleger**
Hi,

I my 1st public contribution for rancher. Just added a turtl cattle template.

Cheers
Frederic

### CODE REVIEW

**Comment by fredleger** [comment]
Sorry i'm not a python CD tests expert and i really don't see what is failing here : 
```
templates = {'data': [{'minimumRancherVersion': u'', 'defaultVersion': u'11.1.580-rancher1', 'files': None, 'upgradeVersionLinks':...', 'license': u'', 'category': u'Logging', 'templateBase': u'', 'catalogId': u'library'}], 'resourceType': u'template'}

    def test_maintainers(templates):
        maintainer = re.compile(r'^([\S]+ ){2,5}<[^@]+@[^@]+\.[^@]+>$')
        for template in templates:
            # Maintainer will soon be a requirement
            # assert template.maintainer
            if template.maintainer:
>               assert maintainer.match(template.maintainer)
E               assert None
E                +  where None = <built-in method match of _sre.SRE_Pattern object at 0x7f810dd3a408>('webofmars <contact@webofmars.com>')
E                +    where <built-in method match of _sre.SRE_Pattern object at 0x7f810dd3a408> = <_sre.SRE_Pattern object at 0x7f810dd3a408>.match
E                +    and   'webofmars <contact@webofmars.com>' = {'minimumRancherVersion': u'', 'defaultVersion': u'1.0.0', 'files': None, 'upgradeVersionLinks': None, 'description': ...rs <contact@webofmars.com>', 'license': u'', 'category': u'Applications', 'templateBase': u'', 'catalogId': u'library'}.maintainer
```
For me the maintener is defined both in the config.yml and the rancher-compose.yml. Can you assist ?

**Comment by matisku** [comment]
I would leave only name and version in rancher-compose.yml. Also match name with name with config.yml

**Comment by cloudnautique** [comment]
@fredleger Thanks!

One question, where does the data get stored? Is this container completely stateless? I see S3 settings but I'm not familiar with this app. If it does have state thats required, it would be best to use a datavolume on the API service. 

Let me know.

File: templates/turtl/0/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    AWS_S3_TOKEN: ${AWS_S3_TOKEN}
+    ADMIN_EMAIL: ${ADMIN_EMAIL}
+    EMAIL_FROM: ${EMAIL_FROM}
+    SMTP_USER: ${SMTP_USER}
+    SMTP_PASS: ${SMTP_PASS}
+    DEFAULT_STORAGE_LIMIT: ${DEFAULT_STORAGE_LIMIT}
+    STORAGE_INVITE_CREDIT: ${STORAGE_INVITE_CREDIT}
+  image: webofmars/turtl-docker:latest
+  stdin_open: true
+  tty: true
```
Comment (cloudnautique) [inline templates/turtl/0/docker-compose.yml]: you'll need to add a labels section here with:
io.rancher.sidekicks: turtl-api-data

**Comment by fredleger** [comment]
Thanks. I got a bit tired when working on it and didn't find what was damm wrong in the compose file :-)

**Pull request merged**